platform/ios/app/MBXFrameTimeGraphView.m

```diff
@@ -0,0 +1,127 @@
+#import "MBXFrameTimeGraphView.h"
+
+const CGFloat EXAGGERATION = 4.f * 1000.f;
+const CGFloat WIDTH = 4.f;
```

Comment:
These two should should be consistent with other constants we have in iosapp - so of the form `MBX...`. But I think we should consider moving to `kMBX...`